cookbooks/llama/python/README.md

```diff
@@ -8,7 +8,7 @@ All GGUF LLaMA models from https://huggingface.co/TheBloke?search_models=gguf sh
 ## Install
 
 ```
-pip install python-aiconfig python-aiconfig-llama
+pip install python-aiconfig aiconfig-extension-llama
```

Comment:
I think this will just create confusion. Why do we need to rename from `python-aiconfig-llama` ?

Reply:
This extension is not called `python-aiconfig-llama`, it is called `python-aiconfig`. We already have that for our core library and this is an extension. We have other extensions follow the format of `aiconfig-extension-XYZ`
<img width="536" alt="Screenshot 2023-12-25 at 16 21 11" src="https://github.com/lastmile-ai/aiconfig/assets/151060367/61a1f507-f06d-4a28-a1bb-9f03148d26c6">

Reply:
Just for extra context, this is where the `python-aiconfig` is defined for our main python library: https://github.com/lastmile-ai/aiconfig/blob/main/python/pyproject.toml#L5

Reply:
Let's push this change. It's very straightforward and doesn't make sense to have been blocked for so long.

If we request changes, please have a fast turnaround on unblocking the diff once comments are addressed.

Reply:
I think this will break the cookbook readme. The extension package is (for better or worse) called `python-aiconfig-llama` and there is no `aiconfig-extension-llama`. The immediate fix for extensions/llama/python/pyproject.toml would be to update the name to the actual existing pypi package, "python-aiconfig-llama". 

Also, even if we updated it to aiconfig_extension_llama and published a new package with that name, I don't think pip install aiconfig-extension-llama would work (underscore vs hyphen). 

What we can do is publish a new package consistent with the other extensions, e.g. aiconfig-extension-llama, and then update the project file and README. 

Isn't it better to postpone cosmetic changes, especially if risky, until editor ships?

Reply:
@jonathanlastmileai this is the same model that every other extension uses. pyproject.toml doesn't allow hyphens, but when you install that package, it is installed as hyphens instead of underscores. This change will work fine.

Please look at other extensions for examples.

Let's ship this.